config/index.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * config loader
+ */
+module.exports = require('../server/utils/dir-auto-loader')(__dirname, false);
```

Comment:
__[eslint]__
* Error - 'module' is not defined. (no-undef)
* Error - 'require' is not defined. (no-undef)
* Error - '__dirname' is not defined. (no-undef)